ui/src/main/resources/MeetingCode/SendInvitationEmail.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.2" reference="MeetingCode.SendInvitationEmail" locale="">
+  <web>MeetingCode</web>
+  <name>SendInvitationEmail</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1455115654000</creationDate>
+  <parent>MeetingCode.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1455169696000</date>
+  <contentUpdateDate>1455169564000</contentUpdateDate>
+  <version>1.1</version>
+  <title/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{include reference="Meeting.Macros" /}}
+{{velocity wiki='false'}}
+#set ($discard = $response.setContentType('application/json'))
+#if ($request.sendNotification == 'true')
+  #set ($meetingDoc = $xwiki.getDocument($request.doc))
+  #set ($message = $services.localization.render('contrib.meeting.notification.body', ["$xwiki.getUserName($xcontext.user)","&lt;strong&gt;$meetingDoc.getDisplayTitle()&lt;/strong&gt;"]))
+
+  #if ("$request.additionalMessage" != '')
```

Comment:
The important part was not the `"` around `$request.additionalMessage`, but the `!` after `$`. It's the velocity specific trick that makes a `null` variable be resolved to an empty string, so it will print ` ` instead of `$request.additionalMessage`. Once you have an empty string, you can compare it with `''`.